tests/test_core.py

```diff
@@ -70,17 +88,16 @@ def test_fourier_filter_err(funcname, err_type, s, n):
         "fourier_gaussian",
     ]
 )
-def test_fourier_filter_identity(funcname, s, dtype):
+def test_fourier_filter_identity(funcname, dtype):
```

Comment:
Should have been renamed. 😱 Fixing in PR ( https://github.com/dask-image/dask-ndfourier/pull/16 ).